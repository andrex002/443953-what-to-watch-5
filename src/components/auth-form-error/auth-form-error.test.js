import React from "react";
import renderer from "react-test-renderer";
import AuthFormError from "./auth-form-error";

it(`Should AuthFormError render correctly`, () => {
  const tree = renderer.create(
      <AuthFormError
        isValidEmail={true}
        isValidPassword={true}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
