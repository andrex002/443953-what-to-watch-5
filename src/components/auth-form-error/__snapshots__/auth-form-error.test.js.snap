// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should AuthFormError render correctly 1`] = `
<div
  className="sign-in__message"
>
  
</div>
`;
